src/reducers/paths.reducer.js

```diff
@@ -43,20 +82,21 @@ export default (state: State = initialState, action: Action) => {
 //
 //
 // Helpers
-const homedir = isWin ? windowsHomeDir : os.homedir();
 // Noticing some weird quirks when I try to use a dev project on the compiled
 // "production" app, so separating their home paths should help.
-export const defaultParentPath =
-  process.env.NODE_ENV === 'development'
-    ? path.join(homedir, '/guppy-projects-dev')
-    : path.join(homedir, '/guppy-projects');
+export const getDefaultProjectHomePath = (
+  state: State = getInitialState().paths || initialState
+) => {
+  return state.defaultHome || initialState.defaultHome;
+};
 
-export const getDefaultPath = (projectId: string) =>
-  `${defaultParentPath}/${projectId}`;
+export const getDefaultPath = (state: State, projectId: string) =>
+  `${getDefaultProjectHomePath(state)}/${projectId}`;
```

Comment:
Right, so now that the project home path lives in Redux state, we need to select it through selectors, not helpers.

This will require some changes, described in detail a bit further down. For this part, though, you'll need to change it to be the following:

```js
// Selectors
export const getProjectHomePath = (state: State) => (
  state.paths.homePath
);

export const getPathForProjectId = (state: State, projectId: string) => {
  const projectPath = state.paths.byId[projectId];

  if (projectPath) {
    return projectPath;
  }

  // If this project doesn't have a specified path, figure out what it should be
  // based on the homePath and projectId
  // TODO: Is this actually necessary?
  const homePath = getProjectHomePath(state);
  
  return `${homePath}/${projectId}`
}
```

That `TODO` comment is because I don't actually think that default value is necessary (and if it is, we should figure out how such a state is possible), but the existing implementation does that, and I don't wanna risk breaking anything in your PR. I can investigate that after it merges.

Right now, you're relying on this method in `create-project.service.js`, so this change will break that call. I have instructions for fixing that below :)